marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/validateMatchingStep.api

```diff
@@ -0,0 +1,14 @@
+{
+    "functionName": "validateMatchingStep",
+    "params": [
+        {
+            "name": "stepName",
+            "datatype": "string"
+        }
+    ],
+    "return": {
```

Comment:
Can you add a description to the "return" here - something like "Returns an array of zero or more warning objects; each object has 'level' and 'message' properties". 

That will also show up in the generated Java code. Ideally, DS API files would let us richly define the types here, but in the absence of that, a description does the job.

Reply:
Done